docs/src/guide/operator_identity.md

```diff
@@ -1,13 +1,33 @@
 # Operator: Identity
 
-## Authorizing the operator `instance_principal`
+## `instance_principal`
 
 [Instance_principal](https://docs.cloud.oracle.com/iaas/Content/Identity/Tasks/callingservicesfrominstances.htm) is an IAM service feature that enables instances to be authorized actors (or principals) to perform actions on service resources. Each compute instance has its own identity, and it authenticates using the certificates that are added to it. These certificates are automatically created, assigned to instances and rotated, preventing the need for you to distribute credentials to your hosts and rotate them.
 
+[Dynamic Groups](https://docs.oracle.com/en-us/iaas/Content/Identity/Tasks/callingservicesfrominstances.htm#:~:text=Concepts-,DYNAMIC%20GROUP,-Dynamic%20groups%20allow) group OCI instances as principal actors, similar to user groups.  IAM policies can then
```

Comment:
This link doesn't seem to load for me with the fragment at the end, as well as the other Dynamic Groups reference below. Maybe a small adjustment is needed, or we could just point to `https://docs.oracle.com/en-us/iaas/Content/Identity/callresources/callingservicesfrominstances.htm`?

Reply:
Same for `Matching Rules` - maybe browser related. I see what you were trying to jump to...

Reply:
What browser are you using?  At least for Chrome and Edge, that link will not jump to that section,
but it the browser will highlight it.

Unfortunately that web page does not have anchor tags for each section, so cannot hyperlink directly to that section

Reply:
You can view a rendered version of the page here: https://github.com/oracle-terraform-modules/terraform-oci-oke/blob/e06b39e6267c74b4235d33b721baacd941fdb40d/docs/src/guide/operator_identity.md

and can try clicking on the links and see what the browser does

Reply:
Yeah, opening a blank page on Firefox for me - maybe this highlighting isn't supported...